src/style.css

```diff
@@ -1,3 +1,147 @@
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  /* height: 406px ; */
+  display: flex;
+  /* flex-wrap: wrap; */
+  flex-direction: column;
+  /* box-sizing: border-box; */
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  display: block;
+  justify-content: center;
+  margin: 32px 19px 40px ;
+  padding: 0;
+  border: 0;
+  /* align-items: center; */
+}
+
+.card__pic {
+  padding: 0;
+  height: 134px;
+  width: 160px;
+}
+
+.card__text {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  margin-bottom: 4px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.card__code {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  margin-bottom: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.card__rating {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.stars {
```

Comment:
better to divide styles, move stars styles to stars.css, card styles to card.css, common styles keep in styles.css